tests/test_manager.py

```diff
@@ -91,6 +91,35 @@ def _normalize_path_arbitrary(fid_manager, path):
     return path
 
 
+def test_migrate_arbitrary_to_local(fid_db_path, jp_root_dir, test_path, test_path_child):
+    arbitrary = ArbitraryFileIdManager(db_path=fid_db_path, root_dir=str(jp_root_dir))
+    arbitrary.con.execute("PRAGMA journal_mode = off")
+
+    id_1 = arbitrary.index(test_path)
+    id_2 = arbitrary.index(test_path_child)
+    del arbitrary
+    local = LocalFileIdManager(db_path=fid_db_path, root_dir=str(jp_root_dir))
+
+    assert local.get_path(id_1) == test_path
+    assert local.get_path(id_2) == test_path_child
+
+
+def test_migrate_local_to_arbitrary(fid_db_path, jp_root_dir, test_path, test_path_child):
+    local = LocalFileIdManager(db_path=fid_db_path, root_dir=str(jp_root_dir))
+    local.con.execute("PRAGMA journal_mode = off")
+
+    id_1 = local.index(test_path)
+    id_2 = local.index(test_path_child)
+    del local
+    # need to normalize the arbitrary content root to produce correct API paths
+    arbitrary = ArbitraryFileIdManager(
+        db_path=fid_db_path, root_dir=str(os.path.normcase(jp_root_dir))
```

Comment:
Good catch, but I think this should be done in the root-dir validator in AFIM, prior to the current conversion of backslashes to forward slashes, since Windows paths are case-insensitive.

Reply:
Sure, let's call `os.path.normcase()` on the contents root only if we determine it to be a local filesystem path via `os.path.isabs()`. In short the logic would look like:

1. Does nothing on POSIX filesystems (since `os.path.normcase()` does nothing in this case)
2. On Windows, only map to lowercase if the contents root is an absolute local path starting with a drive letter.

Reply:
Done. 👍